### DISCUSSION TITLE
How to run the Dashboard on a separate server than my app?

### DISCUSSION THREAD

**Discussion by dejf-js** [errsole/errsole.js#112] [Q&A]
I need to separate the errsole dashboard from my app, so that my app is not connected to the dashboard in any processing like ports or forwards, etc. I want to run the dashboard in an own docker for example and from my app are just the DB inserts executed, but the dashboard is separated?

**Comment by venki91** [comment]
Read this: https://github.com/errsole/errsole.js/discussions/113

@dejf-js Please let me know if the process is not clear. I will write a blog with examples.
